pkg/util/slice/slice.go

```diff
@@ -48,3 +48,13 @@ func ShuffleStrings(s []string) []string {
 	}
 	return shuffled
 }
+
+// StringInSlice checks if str is in slice s
+func StringInSlice(str string, s []string) bool {
```

Comment:
API nits:

I would call this `HasString` and I would flip the args.  That way the caller says `slice.HasString(mySlice, str)`